src/mono/mono/mini/unwind.c

```diff
@@ -404,8 +404,7 @@ mono_unwind_ops_encode_full (GSList *unwind_ops, guint32 *out_len, gboolean enab
 	p = buf;
 
 	loc = 0;
-	l = unwind_ops;
-	for (; l; l = l->next) {
+	for (l = unwind_ops; l; l = l->next) {
```

Comment:
could also move the declaration of `GSList *l` to this for loop
```suggestion
	for (GSlist *l = unwind_ops; l; l = l->next) {
```